pandas/core/series.py

```diff
@@ -420,12 +420,15 @@ def get_values(self):
 
     @property
     def asobject(self):
-        """
+        """DEPRECATED: Use ``astype(object).values`` instead.
```

Comment:
I know "astype(object).values" is the exact replacement, but I would maybe still recommend just "astype(object)". In many cases a Series will just be fine as well, and if the user really wants the array they can do an additional `.values`.

I just want to prevent people using `.values` when it is not needed. Maybe we can mention both as well, although that makes the warning a bit long.

Reply:
don't add ``.values``, that is just adding even more confusion.

Reply:
Ok, recommend to use ``astype(object)``, but still return ``.astype(object).values``, so the API doesn't break.